src/Essentials/src/Screenshot/Screenshot.shared.cs

```diff
@@ -17,17 +17,17 @@ public interface IPlatformScreenshot : IScreenshot
 	{
 #if ANDROID
 		Task<IScreenshotResult> CaptureAsync(Android.App.Activity activity);
-		Task<IScreenshotResult> CaptureAsync(Android.Views.View view);
+		Task<IScreenshotResult?> CaptureAsync(Android.Views.View view);
 #elif IOS || MACCATALYST
 		Task<IScreenshotResult> CaptureAsync(UIKit.UIWindow window);
-		Task<IScreenshotResult> CaptureAsync(UIKit.UIView view);
-		//Task<IScreenshotResult> CaptureAsync(CoreAnimation.CALayer layer, bool skipChildren);
+		Task<IScreenshotResult?> CaptureAsync(UIKit.UIView view);
+		Task<IScreenshotResult?> CaptureAsync(CoreAnimation.CALayer layer, bool skipChildren);
```

Comment:
Capturing a layer was actually used internally and this code was supposed to go public. Since we are not yet GA, we could make this public now - or not if we are wanting to wait.